pkg/cmd/cmd.go

```diff
@@ -0,0 +1,51 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/Azure/gocover/pkg/options"
+	"github.com/spf13/cobra"
+)
+
+// TODO: improve this description.
+var (
+	getLong = `
+Generate unit test diff coverage for go code.
+
+Use this tool to generate diff coverage for go code between two branch.
+In this way to make sure that new changes has a expected test coverage, and won't affect old codes.
+`
+
+	getExample = `# Generate diff coverage report in HTML format, and failure rate should be less than 20%
+gocover --cover-profile=coverage.out --compare-branch=origin/master --format html --failure-rate 20.0 --output coverage.html
+`
+)
+
+// NewGoCoverCommand creates a command object for generating diff coverage reporter.
+func NewGoCoverCommand() *cobra.Command {
+	o := options.NewOptions()
+
+	cmd := &cobra.Command{
+		Use:     "gocover",
+		Short:   "Generate unit test diff coverage for go code",
+		Long:    getLong,
+		Example: getExample,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := o.Run(cmd, args); err != nil {
+				return fmt.Errorf("generate diff coverage %w", err)
+			}
+			return nil
+		},
+	}
+
+	cmd.Flags().StringVar(&o.CoverProfile, "cover-profile", o.CoverProfile, `coverage profile produced by 'go test'`)
+	cmd.Flags().StringVar(&o.CompareBranch, "compare-branch", o.CompareBranch, `branch to compare`)
+	cmd.Flags().StringVar(&o.ReportFormat, "format", o.ReportFormat, "format of the diff coverage report, one of: html, json, markdown")
```

Comment:
default value for this.

Reply:
It has default value for o.ReportFormat and o.CompareBranch in function `NewDiffOptions()`